### PR TITLE
Bookstore: Redux in react

### PR DESCRIPTION
**Pull request by AmanWorku**
# Activities done: 
- [x] Added sample books to the empty array.
- [x] Imported redux `Provider` in the main component file.
- [x] Connected the app with the redux store.
- [x] Used `useSelector` for displaying available books.
- [x] Created a function to add and remove a book using `useDispatch` function.
- [x] Added a function to the `Check Status` button.

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -1,13 +1,77 @@
...
-  <div>
-    <form>
-      <input type="text" placeholder="Title" />
-      <input type="text" placeholder="Author" />
-      <input type="submit" value="Add Book" />
-    </form>
-  </div>
-);
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useState } from 'react';
```
Comment (AmaduKamara) [inline src/components/AddBook.js:2]: - [ ] Writing clean and professional code is the goal for setting up Linters in our project to help check for errors and improper coding and disabling Linter checks defies its purpose. Kindly, try not to disable Linter checks in your code files. Please try to solve the Linter error and resubmit for review. 👍🏽

File: src/components/Book.js [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+      {
+        books.map((book) => (
+          <div key={book.id}>
+            <p>
+              {book.author}
+              {' '}
+              -
+              {' '}
+              {book.title}
+            </p>
```
Comment (AmaduKamara) [inline src/components/Book.js:28]: [Optional Changes]

- [ ] What if you could display the title of the book in an h2 tag and below it, you place the author of the book in a p tag? The title of the book comes first before the author. But this is better handled when you get to the proper styling of the app. 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @AmanWorku,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Used redux correctly 👍🏽
✅ Add and remove book features work as expected 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Let's try not to disable Linter checks in our projects 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/AddBook.js (hunk 1 above)
Comment (AmanWorku) [reply src/components/AddBook.js:2]: @AmaduKamara Thanks sir. Will do accordingly.

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @AmanWorku, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AmanWorku** [comment]
@anagudelogu Thanks you. Wishing you all the best 👍🏼

**Pull request merged**